site/content/docs/01-component-format.md

```diff
@@ -42,13 +42,13 @@ Svelte uses the `export` keyword to mark a variable declaration as a *property*
 
 ---
 
-You can specify a default value, which will be used if the component's consumer doesn't specify a prop.
+You can specify an initial value for a prop. It will be used if the component's consumer doesn't specify the prop on the component. Note that whenever a specified prop is removed by the consumer, the value is set to `undefined` rather than the initial value.
```

Comment:
what about
```suggestion
You can specify an initial value for a prop. It will be used if the component's consumer doesn't specify the prop when instancing the component. Note that whenever a specified prop is removed by the consumer, the value is set to `undefined` rather than the initial value.
```
?

Reply:
I think the right term really is "default initial value". "initial value" alone implies that this will be the starting value, unconditionally.

Furthermore, "if the component's consumer doesn't specify **the prop on the component**" is slightly misleading because `<Cmp x={undefined} />` also results in the default value being used. The following would be more accurate: "if the initial value of the prop is undefined".

That being said, "You can specify a default initial value for a prop." says it all, as far as I'm concerned. The next sentence seems a little superfluous. The real important notice, about which there has been some confusion, is the last one.

Reply:
@rixo you are not wrong, hopefully this would be clear enough?

```suggestion
You can specify a default initial value for a prop. It will be used if the component's consumer doesn't specify the prop on the component (or if its initial value is `undefined`). Note that whenever a prop is removed by the consumer, its value is set to `undefined` rather than the initial value.
```